src/server/index.jsx

```diff
@@ -288,19 +290,41 @@ server
         headers,
       });
 
+      let derivedPageType = 'Unknown';
+
       try {
-        const { service, isAmp, route, variant } = getRouteProps(
-          routes,
-          urlPath,
-        );
-        const data = await route.getInitialData({
+        const {
+          service,
+          isAmp,
+          route: { getInitialData, pageType },
+          variant,
+        } = getRouteProps(routes, urlPath);
+
+        // Set derivedPageType based on matched route
+        derivedPageType = pageType || derivedPageType;
```

Comment:
Is there a unit test for this defaulting behaviour? I can't see one from a skim.

Reply:
Possibly not, will look into this.

Reply:
Good spot @jamesdonoh I have added tests here https://github.com/bbc/simorgh/pull/7058/commits/72680d7a148edb75b454db75ea42b5cdbbf1cead